docs/user/custom-domains.rst

```diff
@@ -0,0 +1,138 @@
+This is a conscious decision as it can be misconfigured in a not easily reversible way.
+For both |org_brand| and |com_brand|, HSTS for custom domains can be set upon request.
+
+We always return the HSTS header with a max-age of at least one year
+for our own domains including ``*.readthedocs.io``, ``*.readthedocs-hosted.com``, ``readthedocs.org`` and ``readthedocs.com``.
+
+.. _Strict Transport Security header: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security
+
+Legacy domains on |com_brand|
+-----------------------------
+
+Some older setups configured a CNAME record pointing to
+``<organization-slug>.users.readthedocs.com``,
+these domains will continue to resolve.
+
+Previously you were asked to add two records,
+this process has been simplified.
+If you have doubts about deleting some of the old records,
+please reach out to :ref:`support <support:user support>`.
+
+Multiple documentation sites as sub-folders of a domain
+-------------------------------------------------------
+
+You may host multiple documentation repositories as **sub-folders of a single domain**.
+For example, ``docs.example.org/projects/repo1`` and ``docs.example.org/projects/repo2``.
+This is `a way to boost the SEO of your website <https://moz.com/blog/subdomains-vs-subfolders-rel-canonical-vs-301-how-to-structure-links-optimally-for-seo-whiteboard-friday>`_.
+
+See :doc:`subprojects` for more information.
+
+Troubleshooting
+---------------
+
+SSL certificate issue delays
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The status of your domain validation and certificate can always be seen on the details page for your domain
+under :guilabel:`Admin` > :guilabel:`Domains` > :guilabel:`YOURDOMAIN.TLD (details)`.
+
+Domains are usually validated and a certificate issued within minutes.
+However, if you setup the domain in Read the Docs without provisioning the necessary DNS changes
+and then update DNS hours or days later,
+this can cause a delay in validating because there is an exponential back-off in validation.
+Loading the domain details in the Read the Docs dashboard and saving the domain again will force a revalidation.
+
+Certificate authority authorization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Certificate authority authorization (CAA) is a security feature that allows domain owners to limit
+which certificate authorities (CAs) can issue certificates for a domain.
+This is done by setting CAA DNS records for your domain.
+
+The readthedocs domains that you'll point your domains to already
+have the proper CAA records.
+
+.. prompt:: bash $, auto
+
+   $ dig +short readthedocs.io CAA
+     0 issue "digicert.com; cansignhttpexchanges=yes"
+     0 issuewild "digicert.com; cansignhttpexchanges=yes"
+     0 issue "comodoca.com"
+     0 issue "letsencrypt.org"
+     0 issuewild "comodoca.com"
+     0 issuewild "letsencrypt.org"
+
+.. prompt:: bash $, auto
+
+   $ dig +short 0acba22b.domains.readthedocs.com CAA
```

Comment:
Who is this domain? Can we use one _without_ a hash?

Reply:
this should be a hashed domain, we can use any hash (`*`)